cluster-autoscaler/cloudprovider/builder/builder_oci.go

```diff
@@ -36,7 +36,7 @@ const DefaultCloudProvider = cloudprovider.OracleCloudProviderName
 func buildCloudProvider(opts config.AutoscalingOptions, do cloudprovider.NodeGroupDiscoveryOptions, rl *cloudprovider.ResourceLimiter) cloudprovider.CloudProvider {
 	switch opts.CloudProviderName {
 	case cloudprovider.OracleCloudProviderName:
-		return oci.BuildOCI(opts, do, rl)
+		return instancepools.BuildOCI(opts, do, rl)
```

Comment:
It _might_ be better to import `instancepools` aliased as "oci" so that we can keep the keep the return `cloudprovider.Build` format. 

i.e. 

```
import (
	oci "k8s.io/autoscaler/cluster-autoscaler/cloudprovider/oci/instancepools"
)
```
...
```
	case cloudprovider.OracleCloudProviderName:
		return oci.BuildOCI(opts, do, rl)
```

Reply:
Added 'oci' alias to Oracle Cloud import to preserve provider naming pattern.